recipe/meta.yaml

```diff
@@ -1,38 +1,43 @@
-{% set version = "2.0.0" %}
+{% set name = "feedgenerator" %}
+{% set version = "2.1.0" %}
 
 package:
-  name: feedgenerator
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/f/feedgenerator/feedgenerator-{{ version }}.tar.gz
-  sha256: 6836d456b8b0edbc5b6d3a42d1be852cebd43d2f28af4ff51789eb295f1860e2
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/feedgenerator-{{ version }}.tar.gz
+  sha256: f075f23f28fd227f097c36b212161c6cf012e1c6caaf7ff53d5d6bb02cd42b9d
 
 build:
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
+    - setuptools >=40.6.0
+    - wheel
+    - poetry-core
     - pip
   run:
-    - python >=3.6
-    - pytz
-    - six
+    - python >=3.7
+    - pytz >=0a
 
 test:
   imports:
     - feedgenerator
-    - feedgenerator.django
-    - feedgenerator.django.utils
```

Comment:
These seem to still be packages https://github.com/getpelican/feedgenerator/blob/main/setup.py#L9, not sure why they were deleted.

Reply:
They are not deleted. Just not necessary to import them all at the testing b/c they are not compiled code.